install_config/configuring_local.adoc

```diff
@@ -189,13 +157,14 @@ The provisioner runs as root to be able to clean up the directories when a PV is
 
 [[local-volume-adding-new-devices]]
 === Adding New Devices
-Adding a new device requires several manual steps:
+Adding new device is semi-automatic. The provisioner periodically checks for new mounts in configured directories. Admin just needs to create a new subdirectory there, mount a device there and allow pods to use the device by applying SELinux label on it:
```

Comment:
s/Adding new device/Adding a new device
s/Admin just needs to/The administrator needs to 
s/mount a device there/mount a device there,
s/by applying SELinux label on it:/by applying the SELinux label:

Reply:
Fixed